### DISCUSSION TITLE
Production tab only shows non-factory production?

### DISCUSSION THREAD

**Discussion by geraldeersteling** [notnotmelon/factorissimo-2-notnotmelon#64] [Q&A] [answered]
First of all I'd like to thank you for all the effort to maintain this mod, it's really appreciated!

Now for the question; when I open the production screen I see very low numbers, even though I have some factories which definitely are producing more then is shown. I also don't see everything that's produced.

I think I do see it when I use " Global statistics", but are these global stats per planet or?

**Comment by notnotmelon** [comment, marked as answer]
Each planet has different numbers for the factory production screen. In factorissimo, the factory floors are technically new planets. This means that each factory production screen only shows the values for the factory floor on that planet. If you want a "merged" view then click the global checkbox.

**Reply by geraldeersteling** [reply]
I had a feeling, but thanks for verifying!
